### PR TITLE
LPD-39729 Remove ID from the object definition class name

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://liferay.atlassian.net/browse/LPD-39729

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by gabrielwas — CHANGES_REQUESTED** [review]
Just one tiny question. 

An amazing job here, @carolmariaabb !

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java [hunk 1]
```diff
@@ -2384,6 +2403,37 @@ private void _validateActive(boolean active, int status)
...
+				ObjectDefinitionConstants.
+					CLASS_NAME_PREFIX_CUSTOM_OBJECT_DEFINITION)) {
+
+			throw new ObjectDefinitionClassNameException.MustStartWithPrefix();
+		}
+
+		ObjectDefinition existingObjectDefinition =
+			objectDefinitionPersistence.fetchByC_C(companyId, className);
+
+		if ((existingObjectDefinition != null) &&
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java:2439]: @gabrielwas yeah, I think we should... I updated the code.

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by gabrielwas — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:report:53896904

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**